static/js/vue-cdr-access/src/components/full_record/folderRecord.vue

```diff
@@ -1,16 +1,77 @@
 <template>
-    <collection-folder-template :record-data="recordData"></collection-folder-template>
+    <div class="content-wrap full_record">
+        <div class="columns browse-top">
+            <div class="column" :class="{restrictedContent: 'is-8'}">
+                <thumbnail :thumbnail-data="recordData"></thumbnail>
+                <h2 :class="isDeleted">
+                    {{ recordData.briefObject.title }}
+                    <span class="item_container_count">{{ displayChildCount }}</span>
+                </h2>
+                <p v-if="fieldExists(recordData.briefObject.added)">
+                    <strong>{{ $t('full_record.date_added') }}: </strong>
+                    {{ formatDate(recordData.briefObject.added) }}
+                </p>
+                <p class="parent_collection" v-if="fieldExists(recordData.briefObject.parentCollectionName)">
+                    <strong>{{ $t('full_record.collection') }}: </strong> <a :href="collectionLink">{{ recordData.briefObject.parentCollectionName }}</a>
+                </p>
```

Comment:
I was looking at how this is implemented in the file and work record pages, and they are both like this:
```
<li v-if="fieldExists(recordData.briefObject.parentCollectionName)">
    <span class="has-text-weight-bold">{{ $t('display.collection') }}: </span>
    <router-link class="parent-collection" :to="parentUrl">{{ recordData.briefObject.parentCollectionName }}</router-link>
</li>
```
I was wondering if we should be using a router-link here too?

Also, it looks like there is already a `parentUrl()` method in `fullRecordUtil` that we can probably use here instead of adding a new collectionLink method (looks like it does the same thing, but with a leading slash).

And I guess it holds true for files, works and folders, but they should always have a parent collection, so we could probably leave off the fieldExists check. It would be a data integrity issue if they didn't.